client/landing/jetpack-cloud/components/date-picker/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { withLocalizedMoment } from 'components/localized-moment';
+import Button from 'components/forms/form-button';
+import DateRangeSelector from 'my-sites/activity/filterbar/date-range-selector';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const DATE_FORMAT = 'YYYY-MM-DD';
+
+class DatePicker extends Component {
+	state = {
+		currentSetting: false,
+	};
+
+	static propTypes = {
+		siteId: PropTypes.number.isRequired,
+		initialDate: PropTypes.instanceOf( Date ).isRequired,
+		onChange: PropTypes.func.isRequired,
+	};
+
+	getFormattedDate = date => this.props.moment( date ).format( DATE_FORMAT );
+
+	getDisplayDate = date => {
+		const word = this.props
+			.moment( date )
+			.calendar()
+			.split( ' ' )[ 0 ];
+		if ( 'Today' === word || 'Yesterday' === word ) {
+			return word;
+		}
+
+		return this.getFormattedDate( date );
+	};
+
+	shuttleLeft = () => {
```

Comment:
What does `shuttleLeft` mean? Are we moving pixels around here?

Reply:
This moves the date into the past.